README.md

```diff
@@ -211,6 +211,36 @@ USB device into the same physical USB port (this method is also used in Linux ke
 Linux USB permissions
 =====================
 
+Kernels after 6.0
```

Comment:
Linux 6.0 paragraph and Kernels before 6.0 have 95% in common.
I would prefer to refactor this to remove Linux 6.0 separate paragraph, and instead add Linux 6.0 specific blocks as outlined below.
We should suggest distro packagers to supply one udev rule file which contains support for both Linux 6.0 and older.